internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -1611,6 +1643,20 @@ public static PubSubMessageHeader getLeaderCompleteStateHeader(LeaderCompleteSta
     return new PubSubMessageHeader(VENICE_LEADER_COMPLETION_STATE_HEADER, val);
   }
 
+  public static PubSubMessageHeader getViewDestinationPartitionHeader(
```

Comment:
Can we move this util function to `ViewUtils` to reduce the length of this file?